configure.ml

```diff
@@ -678,6 +678,22 @@ let operating_system, osdeplibs =
   else
     (try Sys.getenv "OS" with Not_found -> ""), osdeplibs
 
+(** Num library *)
+
+(* since 4.06, the Num library is no longer distributed with OCaml (replaced
+   by Zarith)
+*)
+
+let check_for_numlib () =
+  if caml_version_nums >= [4;6;0] then
+    let numlib,_ = tryrun "ocamlfind" ["query";"num"] in
```

Comment:
This does not respect the `ocamlfindcmd` setting.

Reply:
@maximedenes you requested changes but then merged anyway?

Reply:
Argh!!

Reply:
Funny, I bypassed my own review. @psteckler would you mind fixing this in another PR?